.github/workflows/windows.yml

```diff
@@ -0,0 +1,50 @@
+name: Build Windows
+
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+  pull_request:
+    branches: [ master ]
```

Comment:
Do you know why the PR only shows a `Build Windows / Windows Build (push)` but not a `Build Windows / Windows Build (pull_request)` although both are defined here?

Reply:
because i disabled it before creating the PR